templates/policies/sts_ocm_trust_policy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::896164604406:role/RH-Managed-OpenShift-Installer"
+        ]
+      },
+      "Action": [
+        "sts:AssumeRole"
```

Comment:
we need external id same like user role

Reply:
The external ID should be the Organization ID from AMS

Reply:
fixed